bot/exts/evergreen/game.py

```diff
@@ -136,8 +152,33 @@ def __init__(self, bot: Bot):
 
         self.genres: Dict[str, int] = {}
 
+        self.bot.loop.create_task(self.renew_access_token())
+
         self.refresh_genres_task.start()
 
+    async def renew_access_token(self) -> None:
+        """Refeshes V4 access token 2 days before expiry."""
+        while True:
+            async with self.http_session.post(OAUTH_URL, params=OAUTH_PARAMS) as resp:
+                result = await resp.json()
+                if resp.status != 200:
+                    logger.error(
+                        "Failed to renew IGDB access token, unloading Games cog."
+                        f"OAuth response message: {result['message']}"
+                    )
+                    self.bot.remove_cog('Games')
```

Comment:
I wasn't sure whether to unload the cog, or continue to use the current API token until it expires.